src/photos/ducks/backup/components/BackupActions.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+
+import Button from 'cozy-ui/transpiled/react/Buttons'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import SyncIcon from 'cozy-ui/transpiled/react/Icons/Sync'
+import CheckIcon from 'cozy-ui/transpiled/react/Icons/Check'
+import SpinnerIcon from 'cozy-ui/transpiled/react/Icons/Spinner'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { useBackupActions } from '../hooks/useBackupActions'
+import { useBackupData } from '../hooks/useBackupData'
+
+const BackupActions = () => {
+  const { t } = useI18n()
+
+  const { backupInfo } = useBackupData()
+  const {
+    backupPermissions,
+    prepareBackup,
+    startBackup,
+    stopBackup,
+    requestBackupPermissions
+  } = useBackupActions()
+
+  if (!backupPermissions) return null
+
+  if (backupPermissions.granted) {
+    if (!backupInfo) return null
+
+    const {
+      currentBackup: { status, mediasToBackupCount, totalMediasToBackupCount }
+    } = backupInfo
+
+    if (status === 'running') {
+      return (
+        <div className="u-mt-1 u-flex u-flex-column u-flex-justify-center">
+          <Button
+            label={`${t('Backup.actions.backupInProgress')} ${
```

Comment:
nit: you can pass variable to your translated strings if needed. https://www.npmjs.com/package/node-polyglot